packages/teleterm/src/ui/services/quickInput/parsers.ts

```diff
@@ -136,13 +137,21 @@ export class QuickCommandParser implements QuickInputParser {
   private mapAutocompleteCommandsToSuggestions(
     commands: { name: string; displayName: string; description: string }[]
   ): SuggestionCmd[] {
-    return commands.map(cmd => ({
-      kind: 'suggestion.cmd' as const,
-      token: cmd.displayName,
+    return commands.map(cmd => {
+      const acceptsNoArguments =
+        this.parserRegistry.get(cmd.displayName) instanceof
+        QuickNoArgumentsParser;
```

Comment:
The way it worked before is that `tsh ssh` and `tsh proxy db` had their own parsers assigned to them:

https://github.com/gravitational/webapps/blob/665f7e7f5cc2bb1cb159b5683231d07c3fad7802/packages/teleterm/src/ui/services/quickInput/quickInputService.ts#L63-L70

However, `tsh install` and `tsh uninstall` don't need custom parsers as they don't accept any extra arguments. So I added `QuickNoArgumentsParser` that I use for those two commands.

If that parser is used for the given command, then QuickInputService is not going to append space to the input.